nss-sapphire-elephants/WorkforceManagement
# HM Computer (Issue 8)

Completion of Issue Ticket 8
LIST ALL COMPUTERS
Given a user wants to view all computers
When the user clicks the Computers item in the navigation bar
Then the user will see a list of all Computers

DETAILS
Given the user is viewing all computers
When the user clicks the details link
Then the user should be presented with  all info about the computers

CREATE A COMPUTER
Given user wants create a computer
When the user clicks on the create link
They will be prompted with a form to create a new computer

DELETE A COMPUTER
Given the user wants to delete a computer
When the user clicks on the delete link they will be taken to a conformation page
If they click delete, the item should be deleted ONLY IF IT IS NOT ASSIGNED TO AN EMPLOYEE, 
If it is assigned to an employee, the computer should remain on the screen.

Testing Instructions for Change Made:

git fetch --all
git checkout HMComputer#8
run

See all computers, see details of computer, add computer, and delete computer only if it isn't assigned to an employee already.

Checklist:
My code follows the style guidelines of this project
I have performed a self-review of my own code
I have commented my code, particularly in hard-to-understand areas
My changes generate no new warnings
I have added test instructions that prove my fix is effective or that my feature works
